package.json

```diff
@@ -91,6 +91,8 @@
     "start-server": "node ./server/index.js | node ./server/utils/pino-pg | pino-pretty -c -t",
     "test": "jest",
     "test:integration": "playwright test",
+    "test:integration:debug": "playwright test --debug",
```

Comment:
I'm fine with having these, but I'll also just note you can typically pass arguments to any commands here, e.g. you could (most likely) do this by doing:

```
yarn run test:integration --debug
```